### ISSUE TITLE
RCAN text.yml need be rectified

### ISSUE THREAD

**Issue by SailVR** [xinntao/BasicSR#525]
path:
  pretrain_network_g: 
  strict_load_g: true

# validation settings
val:
  save_img: true
  suffix: ~  # add suffix to saved images, if None, use exp name

  metrics:
    psnr: # metric name, can be arbitrary
      type: calculate_psnr
      crop_border: 2
      test_y_channel: false
    ssim:
      type: calculate_ssim
      crop_border: 2
      test_y_channel: false
